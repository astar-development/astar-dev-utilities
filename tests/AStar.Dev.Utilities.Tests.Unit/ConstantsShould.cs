namespace AStar.Dev.Utilities;

public sealed class ConstantsShould
{
    [Fact(Skip = "Dunno why this is failing on the build server but not locally")]
    public void ContainTheExpectedWebDeserialisationSettingsSetting() =>
        Constants.WebDeserialisationSettings
                 .ToJson()
                 .ShouldMatchApproved();
}