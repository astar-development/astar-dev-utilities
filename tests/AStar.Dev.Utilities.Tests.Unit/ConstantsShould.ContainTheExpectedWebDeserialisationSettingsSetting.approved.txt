{
  "converters": [],
  "typeInfoResolver": null,
  "typeInfoResolverChain": [],
  "allowOutOfOrderMetadataProperties": false,
  "allowTrailingCommas": false,
  "defaultBufferSize": 16384,
  "encoder": null,
  "dictionaryKeyPolicy": null,
  "ignoreNullValues": false,
  "defaultIgnoreCondition": 0,
  "numberHandling": 1,
  "preferredObjectCreationHandling": 0,
  "ignoreReadOnlyProperties": false,
  "ignoreReadOnlyFields": false,
  "includeFields": false,
  "maxDepth": 0,
  "propertyNamingPolicy": {},
  "propertyNameCaseInsensitive": true,
  "readCommentHandling": 0,
  "unknownTypeHandling": 0,
  "unmappedMemberHandling": 0,
  "writeIndented": false,
  "indentCharacter": " ",
  "indentSize": 2,
  "referenceHandler": null,
  "newLine": "\r\n",
  "respectNullableAnnotations": false,
  "respectRequiredConstructorParameters": false,
  "isReadOnly": false
}